cli/lib/kontena/cli/containers/exec_command.rb

```diff
@@ -7,15 +7,61 @@ class ExecCommand < Kontena::Command
     parameter "CMD ...", "Command"
 
     def execute
+      require 'websocket-client-simple'
+      require 'io/console'
+
       require_api_url
       token = require_token
+      cmd = Shellwords.join(cmd_list)
+      base = self
+      ws = connect(token)
+      ws.on :message do |msg|
+        base.handle_message(msg)
+      end
+      ws.on :open do
+        ws.send(cmd)
+      end
+      ws.on :close do |e|
+        exit 1
+      end
+      stdin_thread = Thread.new {
+        STDIN.raw {
+          while char = STDIN.readpartial(1024)
+            ws.send(char)
+          end
+        }
+      }
+      stdin_thread.join
+    end
```

Comment:
Some helper module with `interactive_exec(container)` for service+container exec commands.